docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,25 +12,130 @@ keywords:
+- A What's New doc should include announcements - things we want customers to notice and try out. These could also be notable community contributions where we want to thank a contributor.
+
+When in doubt, ask your nearest PM or EM. Err on the side of _yes, put it in What’s New_.
+
+### Examples of what to include in What’s New
+
+- A new Transformation: [Partition by values](/docs/grafana/latest/whatsnew/whats-new-in-v9-3/#new-transformation-partition-by-values).
+  - This is one of many transformations, but it is brand-new functionality that a user might not notice if they didn’t read the What's New document. What’s New is also a low-effort place to describe some nice use cases and examples for the feature so that users adopt it.
+- The new [Candlestick visualization](/docs/grafana/latest/whatsnew/whats-new-in-v8-3/#candlestick-panel-beta).
+  - This was a beta feature, but still listed in What’s new in order to get the word out and encourage users to try it.
+- All-new Swagger docs for the API. 
+  - This is significant because it makes our documentation much easier to use, and it’s a new place for users to go for help when using our API.
+- [Removing beta labels from several panels](https://github.com/grafana/grafana/pull/58557), which makes then generally available. 
+  - This is a small change code-wise, but a big statement with big customer impact. Customers now know that those plugins are fully supported and recommended for use in production.
+- [New keyboard shortcut](https://github.com/grafana/grafana/pull/61837)
+  - This is a small change, but it brings attention to a feature that has been improved recently and that most people don't know about.
+- [Search improvement for Flame graphs](https://github.com/grafana/grafana/pull/61748)
+  - Fuzzy search! Has to be in the blog post.
+- [Changes to the Prometheus query editor](https://github.com/grafana/grafana/pull/60718)
+  - These are query patterns for the data source that most of our users use.
+
+### Examples of what *not* to include in What’s New
+
+These are important improvements, but are better placed in the changelog than What’s new:
+
+- [A docs update](https://github.com/grafana/grafana/pull/60352)
+  - This update doesn’t require customers to change their behavior - they’ll simply see better instructions the next time they use the docs.
+- [A bug fix related to migrations](https://github.com/grafana/grafana/pull/59438)
+  - This is a bug fix that doesn’t require customer action.
+- [A usability improvement to an existing transformation](https://github.com/grafana/grafana/pull/59074)
+  - Nice fix, but very detailed. Should be in the changelog but not What’s New.
+- [Change regex to accommodate a new branching strategy in Enterprise](https://github.com/grafana/grafana/pull/59429)
+  - This change is invisible to customers.
+
+## Writing guidelines for What's New content
```

Comment:
I think this section might be easier to read if the main bullet item was bolded or something.

Reply:
Good idea, done.